src/components/ChartComponentVS/ChartComponentVS.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useRef } from 'react'
+import Chart from 'chart.js';
+
+const ChartComponentVS = ({ name, stats, statNames, name2, stats2,color, borderColor, color2, borderColor2 }) => {
```

Comment:
name2, color2, border2 are not good names to differentiate between variables, make them more descriptive, firstPokemonName, secondPokemonName